app/models/components/course/course_ability_component.rb

```diff
@@ -17,8 +17,7 @@ def define_permissions
 
   def allow_showing_open_courses
     # TODO: Replace with just the symbols when Rails 5 is released.
-    can [:read, :register], Course, status: [Course.statuses[:published], Course.statuses[:opened],
-                                             'published', 'opened']
+    can [:read, :register], Course, status: [Course.statuses[:published], Course.statuses[:opened]]
```

Comment:
https://github.com/CanCanCommunity/cancancan/commit/5b361a402e4bbe9c03bc0a3ff285aea2e572a917#diff-f6e4fbc507ba12eef3ca788ea825ed02R58
Cancancan also uses the same form in their tests